plugins/feeds/public/zeus_tracker_binaries.py

```diff
@@ -18,29 +18,38 @@ class ZeusTrackerBinaries(Feed):
     }
 
     def update(self):
+
+        since_last_run = datetime.utcnow() - self.frequency
+
         for item in self.update_xml('item',
                                     ["title", "link", "description", "guid"]):
-            self.analyze(item)
-
-    def analyze(self, item):
-        url_string = re.search(r"URL: (?P<url>\S+),",
-                               item['description']).group('url')
-        url_string = url_string.replace(
-            'http://https://', 'https://')  # feed malformed, nasty hack
-        context = {}
-        date_string = re.search(r"\((?P<date>[0-9\-]+)\)",
-                                item['title']).group('date')
-        context['date_added'] = datetime.strptime(date_string, "%Y-%m-%d")
-        context['status'] = re.search(
-            r"status: (?P<status>[^,]+)", item['description']).group('status')
-        context['guid'] = item['guid']
-        context['source'] = self.name
-        try:
-            context['md5'] = re.search(
-                r"MD5 hash: (?P<md5>[a-f0-9]+)",
-                item['description']).group('md5')
-        except AttributeError as e:
-            pass
+            url_string = re.search(r"URL: (?P<url>\S+),",
+                                   item['description']).group('url')
+            url_string = url_string.replace(
```

Comment:
All this logic should go in the `analyze()` method. please keep update as simple as possible